fitbenchmarking/results_processing/tests/test_plots.py

```diff
@@ -6,6 +6,7 @@
 import os
 import unittest
 from tempfile import TemporaryDirectory
+import pandas as pd
```

Comment:
Newline between python libs and 3rd party libs
```suggestion

import pandas as pd
```